### ISSUE TITLE
GetAllCodeReviewsAsyncEnumerable with nullable CodeReviewStateFilter not returning results

### ISSUE THREAD

**Issue by denis417** [JetBrains/space-dotnet-sdk#1]
Hi. I want to call GetAllCodeReviewsAsyncEnumerable() method with null value of state filter. But by default it has other meaning (Opened). Expected "Any" meaning.

And in generated code i found:
`objArray[3] = (object) ((state ?? CodeReviewStateFilter.Opened)?.ToString() ?? "null");`

It seems like there is no way to do what I want right now.

**Comment by maartenba** [comment]
Similar case in the sample app here https://github.com/JetBrains/space-dotnet-sdk/blob/main/src/JetBrains.Space.Samples.Web/Pages/Index.cshtml.cs#L179

Looking at how to properly handle this case.

**Comment by maartenba** [comment]
@denis417 If you are using the CI build NuGet URL, can you try version `0.1.1337-mb-enums.61241.31`, and set state to `null`?

```
await _projectClient.CodeReviews.GetAllCodeReviewsAsyncEnumerable(ProjectIdentifier.Id("..."), ReviewSorting.LastUpdatedDesc, state: null);
```

**Comment by maartenba** [comment]
The fix has been merged to `main` and should show up as a `1.0.0-beta.*` soon.
